Components/Model/Order/LineItem.php

```diff
@@ -77,6 +77,15 @@ public function loadData(\Shopware\Models\Order\Detail $detail)
         $this->setPrice(NostoPriceHelper::format($detail->getPrice()));
         $this->setPriceCurrencyCode(strtoupper($detail->getOrder()->getCurrency()));
 
+        $articleDetail = Shopware()
+            ->Models()
+            ->getRepository('Shopware\Models\Article\Detail')
+            ->findOneBy(array('number' => $detail->getArticleNumber()));
+
+        if ($articleDetail) {
```

Comment:
We are using `empty()` check in Shopware usually so perhaps change this one as well to keep things consistent

Reply:
Fixed, thanks 😀